bin/linters.sh

```diff
@@ -81,11 +81,5 @@ function check_licenses() {
     echo 'Licenses OK'
 }
 
-ensure_pilot_types
-check_licenses
 install_golangcilint
 run_golangcilint
-run_adapter_lint
-run_test_lint
-run_helm_lint
-check_grafana_dashboards
```

Comment:
I don't we should be removing these.

Reply:
@jeffmendoza can you provide details on `check_grafana_dashboards` ?

Reply:
[fix_datasources.sh](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/grafana/fix_datasources.sh) should be run on dasbboards. This lint checks that it is run. If it is failing it needs to be run, and the results checked in.

Reply:
Sorry, I didn't mean to remove any of these. Just a leftover of my testing.